pubspec.yaml

```diff
@@ -1,9 +1,9 @@
 name: fixnum
-version: 0.10.5
+version: 0.10.6
 author: Dart Team <misc@dartlang.org>
 description: Library for 32- and 64-bit signed fixed-width integers.
 homepage: https://github.com/dart-lang/fixnum
 dev_dependencies:
   test: ^0.12.0
 environment:
-  sdk: ">=0.8.10+6 <2.0.0"
+  sdk: ">=0.8.10+6 <2.0.0-dev.infinity"
```

Comment:
Updated lower bound of SDK to `>=1.8.3` – since this is where the `^` syntax is valid